ethpm_types/utils.py

```diff
@@ -89,6 +89,34 @@ def _sort(value: Any) -> Any:
     return json.dumps(sorted_settings, separators=(",", ":"), sort_keys=True)
 
 
+def parse_signature(sig: str) -> tuple[str, list[tuple[str, str, str]]]:
+    """
+    Parse an event or function signature into name and inputs
+
+    Args:
+        sig (str): The string signature to parse.
+
+    Returns:
+        A tuple of (name, inputs) where inputs is a list of tuples of (type, indexed, arg name).
+    """
+    name, remainder = sig.split("(")
```

Comment:
would be helpful to have outputs too (signature contains outputs)

Reply:
Outputs are [not standard in signatures](https://docs.soliditylang.org/en/develop/abi-spec.html#function-selector) and not something I've seen before in the wild.

Reply:
There's a difference between `selector` and `signature`, I would say that in this library `signature` contains the ouput, so it's important that we can handle that (it should round trip `MethodABI.from_signature(method_abi.signature) == method_abi`)

Reply:
Added in 8a67d25